pkg/kubectl/cmd/run.go

```diff
@@ -478,8 +478,9 @@ func handleAttachPod(f *cmdutil.Factory, c *client.Client, ns, name string, opts
 	opts.Client = c
 	opts.PodName = name
 	opts.Namespace = ns
+	stderr := opts.Err
 	if err := opts.Run(); err != nil {
-		fmt.Fprintf(opts.Err, "Error attaching, falling back to logs: %v\n", err)
+		fmt.Fprintf(stderr, "Error attaching, falling back to logs: %v\n", err)
```

Comment:
Shall we add a check here to make sure stderr is not `nil` or we're good?

cc @ncdc why set the `opts.Err` to `nil` in `opts.Run()`? It's surprising to the callers IMO.